messageService/message_test.go

```diff
@@ -0,0 +1,104 @@
+package messageService
+
+import (
+	"testing"
+	"time"
+	"fmt"
+)
+
+var (
+	Type_ = "messageService"
+	Command = "send message"
+	Sender = 1
+	Receiver = 2
+	CurrentTime = time.Now
+	Auth = "token"
+	Text = "msg from user 1 to user 2"
+	byteMessage = []byte(`{"header":{"type":"messageService","command":"send message","sender":1,"receiver":2,"time":"2017-07-15T01:11:04.134765+03:00","auth":"token"},"body":{"text":"msg from user 1 to user 2"}}`)
+	byteHeader = []byte(`{"type": "message", "command": "sentMessage", "sender": 1, "receiver": 2, "time": "hereAndNow", "auth": "token"}`)
+	byteBody = []byte(`{"text": "msg from user 1 to user 2"}`)
+	)
+
+func TestMarshalMessage(t *testing.T) {
+	header := MessageHeader{Type_, Command, Sender, Receiver, CurrentTime() , Auth}
+	body := MessageBody{Text}
+	message := Message{header, body}
+	mrsMsg, err := MarshalMessage(message)
+	if err == nil {
+		fmt.Println("TestMarshalMessage: PASS, Error-> ", err)
```

Comment:
In tests you should use `t.Error()` or `t.Fatal()`, not `fmt.Println()`.
And try to avoid excess output like "=========...==="